### ISSUE TITLE
set_priors bug

### ISSUE THREAD

**Issue by macartan** [macartan/gbiqq#52]
alpha priors misplaced below 
```
model <- make_model("X -> Y <- Z")  %>%
	set_restrictions(causal_type_restrict = "(Y[X=1, Z=.] < Y[X=0, Z=.])")  %>%
	set_priors(alphas = list(X = c(X0 = 100, X1=100), Y = c(Y0001 = 20)))

> model

 DAG: 
  parent children
1      X        Y
2      Z        Y


 lambda priors: 
   X.X0    X.X1    Z.Z0    Z.Z1 Y.Y0000 Y.Y0100 Y.Y1100 Y.Y0001 Y.Y0101 Y.Y1101 Y.Y0011 Y.Y0111 
      1       1       1       1       1       1       1       1       1       1       1       1 
Y.Y1111    <NA>    <NA>    <NA> 
      1     100     100      20 
```

**Comment by lilymedina** [comment]
this is solved in the PR [#48](https://github.com/macartan/gbiqq/pull/48#issue-289031380)

**Comment by lilymedina** [comment]
now merged into master

> model <- make_model("X -> Y <- Z")  %>%
+     set_restrictions(causal_type_restrict = "(Y[X=1, Z=.] < Y[X=0, Z=.])")  %>%
+     set_priors(alphas = list(X = c(X0 = 100, X1=100), Y = c(Y0001 = 20)))
Generated expanded expression:
(Y[X=1, Z=0] < Y[X=0, Z=0] | Y[X=1, Z=1] < Y[X=0, Z=1])
> model

```
 DAG: 
  parent children
1      X        Y
2      Z        Y


 lambda priors: 
   X.X0    X.X1    Z.Z0    Z.Z1 Y.Y0000 Y.Y0100 Y.Y1100 Y.Y0001 Y.Y0101 Y.Y1101 Y.Y0011 Y.Y0111 Y.Y1111 
    100     100       1       1       1       1       1      20       1       1       1       1       1 
```
